synapse/config/oidc_config.py

```diff
@@ -158,6 +161,10 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
           #
           #skip_verification: true
 
+          # if user already exists, add oidc token to that account instead of failing. Defaults to false.
+          #
+          #merge_with_existing_users: false
```

Comment:
https://github.com/matrix-org/synapse/blob/develop/docs/code_style.md#configuration-file-format says:

> For boolean (on/off) options, convention is that this example should be the opposite to the default (so the comment will end with "Uncomment the following to enable [or disable] ."

please could you follow the conventions.